apps/docs/content/guides/database/connection-management.mdx

```diff
@@ -0,0 +1,98 @@
+---
+id: 'connection-management'
+title: 'Connection management'
+description: 'Managing connections'
+subtitle: 'Using your connections resourcefully'
+---
+
+## Connections
+
+Every [Compute Add-On](/docs/guides/platform/compute-add-ons) has a pre-configured direct connection count and Supavisor pool size. This guide discusses ways to observe and manage them resourcefully.
+
+## What is a pool
+
+When clients (backend servers) connect to PostgreSQL or Supavisor, they use the PostgreSQL Wire Protocol. Unlike HTTP requests, these connections are stateful and persistent, involving a costly startup but becoming efficient once established.
+
+Clients may create several active connections with Postgres, reducing the overhead of establishing connections between queries.
+
+## The pools in Supabase
+
+Supabase servers only create connections when needed, closing idle ones to conserve resources. If you're not using a service, no connections will be made. They are configured to not over-consume the database connections.
```

Comment:
I recommend pulling this out completely b/c it's confusing and doesn't really help me understand Supavisor in any meaningful way.

Reply:
The section was originally much longer, discussing connection management for PostgREST, Realtime, etc. However, after talking to Steve, I was discouraged from adding it. Nobody wanted users to toy around with already optimized configs.

This portion was a remnant of the original section and was intended to minimize connection management anxiety around the Supabase Client. Without the extra context, I guess it seems out of place. I'll remove it